97450 Quote export not working / PO on G/L

tyndmar opened the pull request.

Commit:
97749 PU11Invalid GL Error When Entering Item on PO

program changed:
po/d-poordlN.w - added completion of GL actnum when adding/changing an item on a po-line and actnum has not yet been set

Commit:
97450 FW: PF3 VCM Export not Working

expanded fields in est/rd-qhexp.w (EQ1 export function)

jay-advantzware reviewed (commented):
Profit is / $0.00 is likely still going to fail - need to handle exceptions and make sure they do not cause an error if we use any format for data.
Ideally a procedure that we pass (pProfit(->>,>>9.99%) would return a valid value or the upper/lower limit.  For example -100000000000 would return -99,999.99% and same for high value.  If we have this global procedure, then anytime we get this error, we can add procedure to handle it.  Otherwise, we end up putting little logic checkers everywhere.

jay-advantzware reviewed (commented):
RUN get-itemfg-gl (INPUT cocode, INPUT SELF:screen-value, OUTPUT cActNum).
- There is an NK1 = Ap GL# that needs to be used here, so if blank, should not prefill it as it likely is blank in the source defined.

Commit:
97450 FW: PF3 VCM Export not Working

It appears that values are left-shifted when profit is undefined (zero cost), so setting to max-values in this event.

Commit:
97450 PF3 VCM Export not Working

Programs changed:
est/rd-qhexp.w
po/VendCostExp.w
removed format clauses from STRING(x) where x is an integer or decimal

jay-advantzware committed:
Delete d-poordlN.w
